Marlin/stepper.cpp

```diff
@@ -48,6 +48,7 @@ block_t* current_block;  // A pointer to the block currently being traced
 // Variables used by The Stepper Driver Interrupt
 static unsigned char out_bits = 0;        // The next stepping-bits to be output
 static unsigned int cleaning_buffer_counter;
+static unsigned char last_extruder = 0;
 
```

Comment:
This is used only in `trapezoid_generator_reset()`. So does not need to be global.. Declare it in the function.